Assets/Scripts/PlayerShooting.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerShooting : MonoBehaviour
+{
+    public GameObject arrowPrefab;
+    public float speed = 100f;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+     
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetMouseButtonDown(0))
+        {
+            GameObject arrowObject = Instantiate(arrowPrefab) as GameObject;
+            arrowObject.transform.position = transform.position + transform.forward;
+            arrowObject.transform.forward = transform.forward;
+            arrowObject.GetComponent<Rigidbody>().AddForce(transform.forward * speed);
```

Comment:
Oh, I just realized you should use the AddForce should also have ForceMode.Impulse because it's not a continuously applied force. It's not super important though. It just makes it so you're force doesn't need to be a huge number. I'll still accept the pull request